mslice/models/slice/mantid_slice_algorithm.py

```diff
@@ -12,13 +13,19 @@
 KB_MEV = constants.value('Boltzmann constant in eV/K') * 1000
 HBAR_MEV = constants.value('Planck constant over 2 pi in eV s') * 1000
 E_TO_K = np.sqrt(2*constants.neutron_mass)/HBAR_MEV
+E2L = 1.e23 * constants.h**2 / (2 * constants.m_n * constants.e)  # energy to wavelength conversion E = h^2/(2*m_n*l^2)
+crystal_structure = {'Copper': ['3.6149 3.6149 3.6149', 'F m -3 m', 'Cu 0 0 0 1.0 0.05'],
+                     'Aluminium': ['4.0495 4.0495 4.0495', 'F m -3 m', 'Al 0 0 0 1.0 0.05'],
+                     'Niobium': ['3.3004 3.3004 3.3004', 'I m -3 m', 'Nb 0 0 0 1.0 0.05'],
+                     'Tantalum': ['3.3013 3.3013 3.3013', 'I m -3 m', 'Ta 0 0 0 1.0 0.05']}
+MEV2J = 1 / (4000 * np.pi * constants.epsilon_0) # coulomb constant / 1000
```

Comment:
I don't think `MEV2J` is used anywhere - maybe it can be removed?

Also, this is not the right conversion factor - `1/(4* np.pi * constants.epsilon_0)` is indeed the "Coulomb constant" which comes from Coulomb's Law, but actually what we need is the "elementary electronic charge" which is 1.602e-19 Coulombs (the unit) - and denoted by `constants.e` or `constants.elementary_charge`. Sorry about the confusion but amongst physics neutron scatterers this is often refered to (incorrectly) as the "Coulomb constant" (because of the unit).